src/components/CSSmodules/footer.css

```diff
@@ -0,0 +1,23 @@
+.magicians {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-size: 30px;
+}
+
+.links {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-size: 15px;
+}
+
+.thesect {
+  display: flex;
+  justify-content: space-between;
+  background-color: rosybrown;
+  padding: 1%;
+  flex-wrap: wrap;
+  align-items: baseline;
+}
```

Comment:
- [ ] [OPTIONAL] It will be better if you add these properties to the footer to make it fixed at bottom of the screen.
```CSS
  position: fixed;
  bottom: 0;
  width: 100%;
```